book/src/getting-started/4-cmake-integration.md

```diff
@@ -116,6 +116,12 @@ $ build/examples/qml_minimal/example_qml_minimal
 
 You should now see the two Labels that display the state of our `MyObject`, as well as the two buttons to call our two Rust functions.
 
+### Windows with MSVC
+
+If you're building qt-cxx on Windows using MSVC generator, you need to add the `-DCMAKE_MSVC_RUNTIME_LIBRARY=MultiThreadedDLL` flag to the cmake command when building with the `Debug` configuration. This flag is necessary to ensure that the correct C Runtime Library is used. Then you can build using `cmake --build build --config Debug`.
```

Comment:
cxx-qt not qt-cxx :-) And generally if it's not the crate name and being used as the name of the project I think we are trying to refer to it as CXX-Qt.